nilearn/plotting/html_connectome.py

```diff
@@ -27,8 +27,33 @@ def _prepare_line(edges, nodes):
     return path_edges, path_nodes
 
 
+def _prepare_colors(marker_color, path_nodes):
+    marker_colors, marker_colorscale = None, None
+
+    if isinstance(marker_color, str) and marker_color == 'auto':
+        marker_colors = np.linspace(0, 100, len(path_nodes)).astype("uint8")
+        marker_colors = marker_colors[path_nodes].tolist()
+        marker_colorscale = 'Viridis'
+    elif isinstance(marker_color, str):
+        color_as_hex = mpl_colors.to_hex(marker_color)
+        marker_colorscale = [[0, color_as_hex], [1, color_as_hex]]
+        marker_colors = [0] * len(path_nodes)
+    elif isinstance(marker_color, list):
+        if len(marker_color) != np.max(path_nodes) + 1:
+            raise ValueError("Number of colors provided should match the number of nodes")
+        # Colorscale values must be between 0 and 1
+        marker_colorscale = [
```

Comment:
I have the impression it would be more natural to pass a list of colors (one for each node) in this case than to hack a colorscale. WDYT?

Reply:
Indeed. For the `auto`, how would you sample the colors? Maybe by using the `Set2` colormap as in `plot_connectome`?

Reply:
yes let's do the same as plot_connectome